test/import/FSHImporter.ValueSet.test.ts

```diff
@@ -1246,6 +1285,32 @@ describe('FSHImporter', () => {
         expect(vs.rules).toHaveLength(1);
         assertInsertRule(vs.rules[0] as Rule, '', 'MyRuleSet');
       });
+
+      it('should parse an insert rule at a concept', () => {
+        const input = leftAlign(`
+        ValueSet: ZooVS
+        * ZOO#hippo insert DesignationRules
```

Comment:
In practice, this would be invalid, right? Because you need to introduce the code first before you can use it as a context for insert?

Reply:
Yes, this would produce an error during export, since it wouldn't be able to find the `ZOO#hippo` concept anywhere in the compose lists. I can add the concept component in so that this is a more valid scenario.